website/markdown/docs/links.mdx

```diff
@@ -73,3 +73,9 @@ import {
 - [Manifesto](/docs/contribution/manifesto/)
 - [Performance](/docs/contribution/performance/)
 - [Generation Pipeline](/docs/contribution/generation-pipeline/)
+- [Website](/docs/contribution/website/)
+
+#### Stats
+
+- [About](/docs/stats/about/)
```

Comment:
I'd move this one to the "usage" section and rename it to "Usage statistics". I don't think there's enough context around statistics to justify its own section.

Reply:
Moved it on `usage`, but kept the name since `"Usage statistics"` doesn't sound clear to me